app/breakpoints.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
Please align the names and sizes with https://github.com/ilios/common/blob/master/app/styles/ilios-common/mixins/media.scss and add a comment to this file that the two are linked.  I wonder if we can add this file to common and have it exported. We can investigate that in a different place though.

Looks like a bit more active voice here so `for-laptop-and-up` could just be `laptopAndUp`.

Reply:
👍